note/技術/persuade 對玩家.md

```
@@ -0,0 +1,101 @@
+script: scripts/character/agent.gd, scripts/character/player.gd, scripts/ui/persuade_dialog.gd
+updated: 2026-08-21
+---
+
+# persuade 對玩家
+
+NPC 對玩家發起 `persuade` 時的 Y/N 彈窗流程（issue #305）。依賴 #227（`persuade`
+核心機制）與 #415（waypoint 導引元件），兩者都已落地。
+
+## 為什麼不是 `_pending_persuade` 那條路
+
+#227 的核心機制是「待回應事實句注入下一輪決策，被說服者的模型自己輸出
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)